### ISSUE TITLE
test suite

### ISSUE THREAD

**Issue by neok-m4700** [wouterboomsma/eigency#9]
Hi,

I have a failing test with numpy 1.12.0 and python 3.6.0

```bash
$ python run_tests.py
._frozen_importlib:205: ImportWarning: can't resolve package from __spec__ or __package__, falling back on __name__ and __path__
E........................
======================================================================
ERROR: test_function_type_char (__main__.TestEigency)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "run_tests.py", line 166, in test_function_type_char
    mat_out = eigency_tests.function_type_uint8(mat_in)
  File "eigency_tests/eigency_tests.pyx", line 140, in eigency_tests.function_type_uint8 (eigency_tests/eigency_tests.cpp:3017)
    return ndarray(_function_type_char(FlattenedMap[Array, char, Dynamic, Dynamic](array)))
  File "eigency/conversions.pyx", line 164, in eigency.conversions.ndarray_copy_char_F (eigency/conversions.cpp:6998)
  File "<...>/lib/python3.6/site-packages/numpy/core/numeric.py", line 531, in asarray
    return array(a, dtype, copy=False, order=order)
ValueError: invalid literal for int() with base 10: '\x01'

----------------------------------------------------------------------
Ran 26 tests in 0.010s

FAILED (errors=1)
```

Is it a known issue ?

**Comment by wouterboomsma** [comment]
Hi, 

Thanks for the report. I just reproduced it on my system. I'll try to fix this as soon as possible.

**Comment by wouterboomsma** [comment]
Fixed in 797eb1250dfa9c8f9b1428b0ddecc2ad0f35c9eb. I'll put out a new release as soon as I fix the other issue you raised. Closing this now - but feel free to open again if you still experience this problem.
